File: Python/Warmup - 2/last2.py
def last2(str):
    
    if len(str) <= 2:
        
        return 0
    
    substring = str[-2:]
    
    count = 0
    
    for i in range(len(str) - 2):
        
        test = str[i:i+2]
        
        if test == substring:
            
            count += 1
            
    return count

File: Java/Warmup - 2/last2.java
public int last2(String str) {

    int count = 0;
  
    if(str.length() > 2) {
    
        String lastString = str.substring(str.length()-2);
    
        for(int i=0;i<str.length() - 2; i++) {
    
            if(str.substring(i,i+2).equals(lastString)) {
        
                count++;
        
            }
    
        }
    
    }
    
    return count;

}